Shivani_Bansal/calculator/Calculator.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Calculator</title>
+    <link rel="stylesheet" href="Calculator.css">
```

Comment:
Folder structure missing 
all static assets must be under /assets
for eg
/Project
---/assets
------/css
---------Calculator.css
------/scripts
---index.html